core/hybrisadaptor.cpp

```diff
@@ -453,6 +443,7 @@ void HybrisAdaptor::stopSensor()
         entry->setIsRunning(false);
         running_ = false;
         shouldBeRunning_ = false;
+        inStandbyMode_ = false;
```

Comment:
This particular line only happens when the last sensor gets stopped. The upper sensor object will keep it's alwaysOn/standbyOverride property intact and will start the sensor with that property.